coreax/kernel.py

```diff
@@ -368,11 +374,17 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class LaplacianKernel(Kernel):
-    """
+    r"""
     Define a Laplacian kernel.
 
-    :param length_scale: Kernel ``length_scale`` to use
-    :param output_scale: Output scale to use
+    Given :math:`\lambda =`'length_scale' and :math:`\rho =`'output_scale', the
+    Laplacian kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = \rho \exp(\frac{||x-y||_1}{2 \lambda^2})`  where
+    :math:`||\cdot||_1` is the :math:`L_1`-norm.
+    .
+    :param length_scale: Kernel smoothing/bandwidth parameter, :math:`\lambda`
+    :param output_scale: Kernel normalisation constant, :math:`\rho`
```

Comment:
Fixes param rendering error and improves equation rendering
```suggestion
    :math:`k(x, y) = \rho * \exp(\frac{||x-y||_1}{2 \lambda^2})`  where
    :math:`||\cdot||_1` is the :math:`L_1`-norm.

    :param length_scale: Kernel smoothing/bandwidth parameter, :math:`\lambda`
    :param output_scale: Kernel normalisation constant, :math:`\rho`
```